src/network/protocol/BlockHeader.zig

```diff
@@ -0,0 +1,42 @@
+const std = @import("std");
+
+pub const BlockHeader = struct {
```

Comment:
In #127 @gdnathan added it to `src/types/block.zig` I think it is a better place.
I looks like both your PR will conflict but lets still work on those